agent/command.go

```diff
@@ -114,6 +115,33 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 			tc.setCurrentIdleTimeout(nil)
 		}
 
+		if cmd.Name() == "manifest.load" {
+			manifestLoadCalled = true
+		}
+
+		if cmd.Name() == "git.get_project" && !manifestLoadCalled {
```

Comment:
Hmmm, I'm not sure. I think the if the user does it again it may be intentional? 
cc: Ryan

Reply:
The hypothesis of this change is that calling `get_project` without `manifest.load` is a bug in a project's config and that this bug prevents proper module pinning.

If the project is calling `manifest.load` *after* `get_project` then either it is a bug, or they are intentionally lying to later parts in the project about the `${*_rev}` expansions that were used during checkout or something. The workaround for them in that case is to simply not use the expansions and to use the original branch spec used in the module definition.

So TLDR: Reasonable question, but if we're going to consider this case, I'd advocate to error if the user attempts to call `manifest.load` after `get_project`: it's a sign that evergreen (the team)'s understanding of the module lifecycle is very different from the project's.

Reply:
Erroring seems strong since it could break existing behavior but it could make sense to no-op and/or provide a warning that this may not be correct (not sure how easy it would be to do this though, possibly a second ticket)

Reply:
Ya not erroring is fine.